src/pages/[lang]/[model]/[slug].astro

```diff
@@ -0,0 +1,58 @@
+---
+import type { GetStaticPaths } from "astro";
+import Container from "@components/Container.astro";
+import { getTranslations } from "@i18n/server";
+import Layout from "@layouts/Layout.astro";
+import { getEntry } from "astro:content";
+import { getCollection } from "astro:content";
+import config from "@config";
+import RecordDetail from "@components/RecordDetail/index.astro";
+
+// TODO: I haven't figured out how to get this to be set by config or env variable
+export const prerender = true;
+
+const { t } = await getTranslations(Astro.currentLocale);
+
+const { model, slug } = Astro.params;
+
+const lang = Astro.currentLocale;
+
+let data: any;
+
+if (prerender) {
+  // @ts-ignore
+  const entryResponse = await getEntry(model, slug);
+  data = entryResponse.data;
+} else {
+  const apiUrl = new URL(`/api/${model}/${slug}.json`, Astro.url.origin);
```

Comment:
if we do version the API, should this be `/api/${version}/${model}/${slug}.json`?